auto_round/autoround.py

```diff
@@ -523,7 +528,7 @@ def set_layerwise_config(self, layer_config):
         """
         layers_in_blocks = get_layer_names_in_block(self.model, self.supported_types, self.quant_block_list)
         keys = ["data_type", "bits", "group_size", "sym", "scale_dtype", "act_bits", "act_group_size", "act_sym",
-                "act_dynamic", "act_data_type"]
+                "act_dynamic", "act_data_type", "super_bits", "super_group_size"]
```

Comment:
Better not dump too much information to the model. If double quant is not enabled, can we remove this info? The same applies to activation.